### ISSUE TITLE
RCL A-E and RCL (i) disable stack lift

### ISSUE THREAD

**Issue by whuyse** [zathras/jrpn#81]
**JRPN 15C?**

**Describe the bug**
RCL followed by a matrix letter A-E or (i) (when register I contains a matrix descriptor) disables stack lift, i.e. if the next entry is a number or another RCL, it will _overwrite_ whatever the result of RCL A-E or (i) was.

**Debug Log**

**To Reproduce**
**setup**:
make sure USER mode is off
1
ENTER
f DIM A
f MATRIX 1
-5
STO A
**now do**:
2
RCL A
3
X<>Y
It will return 2, while it should return -5, the result of RCL A

**Expected behavior**
RCL A-E and RCL (i) must work like any other RCL - e.g RCL 1 or RCL I, and enable stack lift.

**Platform:**
JRPN 15C Version 2.1.10 web running on Windows 10 Enterprise under Chrome 122.0.6261.95

**Additional context**

**Comment by zathras** [comment]
@whuyse Thanks!  This was a tricky one to get right.  The 15C has the behavior where a matrix operation like this can be cancelled by holding the button.  Implementing an operation that can be canceled  doesn't play nice with the way most other operations work.

Do you think you'll be finding more 15C issues in the near future?  If you're still exploring, I'd like to hold off making a release until things have settled down.

Also, would you be willing to try out the fix?  If so, I can make a build for you if needed.  It looks like you're running from the web site (jrpn.jovial.com/run15) -- is that right?  Do you have a way to run a beta release on one of the other platforms?  If not, it's not a big deal; I can just publish the beta to the web.

**Comment by whuyse** [comment]
Out of all simulators, I selected yours because
- it really looks amazing
- it allows import/export of programs
- it is web-based.
I only have a corporate laptop, so I can't install unauthorized software, and I have an iPhone.. so no other option is available for me for testing - which of course I'm willing to do.
I'm only a recent convert to the 15C, so there's no telling if or when I may find other issues.. I have yet to explore the realm of complex numbers, for instance. For now, it's matrix-related programs, and I do tend to use the full spectrum of related instructions ;-)
There's no need to rush out a release, I'd say. Both issues I reported have simple workarounds. I do have a suggestion for the printout, I'll open a new issue for it.
Cheers, Werner
